src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -6,3 +6,4 @@ override Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.ToString() ->
 
 Microsoft.CodeAnalysis.ITypeSymbol.IsExtension.get -> bool
 Microsoft.CodeAnalysis.TypeKind.Extension = 14 -> Microsoft.CodeAnalysis.TypeKind
+Microsoft.CodeAnalysis.ITypeSymbol.ExtensionParameter.get -> Microsoft.CodeAnalysis.IParameterSymbol?
```

Comment:
Do we have public API approval or tracking issue?

Reply:
For feature branch, we can proceed without API approval. I'm tracking this as part of list of work still remaining for the feature